tools/test-template/test-template-mac/Utilities/DirectoryFinder.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace Xamarin.Tests.Templating
+{
+	static class DirectoryFinder
+	{
+		public static string FindRootDirectory ()
```

Comment:
The Configuration class already has API that does the same thing:

https://github.com/xamarin/xamarin-macios/blob/f389c03970b0b48b3300340d2cd54a0af6f44fa8/tests/common/Configuration.cs#L329-L333